Roles/Neutral/Berserker.cs

```diff
@@ -161,7 +164,7 @@ public override bool OnCheckMurderAsKiller(PlayerControl killer, PlayerControl t
                 if (player == killer) continue;
                 if (player == target) continue;
 
-                if (Utils.GetDistance(killer.transform.position, player.transform.position) <= Bomber.BomberRadius.GetFloat())
```

Comment:
What does this do?

Reply:
I added a check for if can kill teamate is true or false.

Reply:
Nvm this code is foul

Reply:
Fixed, idk what I was thinking making this lol